qualtran/serialization/sympy.py

```diff
@@ -25,6 +25,8 @@ def _get_sympy_function_type(expr: sympy.Basic) -> int:
 
     This method converts a sympy function to its sympy_pb2.Function enum representation.
     """
+    import sympy.codegen.cfunctions
+
```

Comment:
Why the import? We don't seem to be using it?

Reply:
I added `sympy.codegen.cfunctions.log2` in https://github.com/quantumlib/Qualtran/pull/1088, and the tests passed because `sympy_test.py` imported it. But otherwise `import sympy` itself does not load `sympy.codegen`.